src/MOI_wrapper.jl

```diff
@@ -431,5 +433,79 @@ end
     end
     # qmatind is ::Vector{Cint}, so we convert back to ::Vector{Int}.
     return sparse(Int.(qmatind), qmatcol, qmatval)
- end
+end
+
+function compute_conflict(model::Optimizer)
+    model.conflict = refineconflict(model.inner)
+    return
+end
 
+"""
+    ConflictStatus()
+
+An optimizer attribute indicating the status of the last computed conflict.
+"""
+struct ConflictStatus <: MOI.AbstractOptimizerAttribute  end
+MOI.is_set_by_optimize(::ConflictStatus) = true
+
+function MOI.get(model::Optimizer, attribute::ConflictStatus) 
+    if model.conflict == nothing
+        return MOI.OPTIMIZE_NOT_CALLED
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_MINIMAL
+        return MOI.OPTIMAL
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_FEASIBLE
+        return MOI.INFEASIBLE
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_CONTRADICTION
+        return MOI.OTHER_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_DETTIME_LIM
+        return MOI.TIME_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_IT_LIM
+        return MOI.ITERATION_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_MEM_LIM
+        return MOI.MEMORY_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_NODE_LIM
+        return MOI.NODE_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_OBJ_LIM
+        return MOI.OBJECTIVE_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_TIME_LIM
+        return MOI.TIME_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_USER
+        return MOI.OTHER_LIMIT
+    end
+end
+
+function MOI.supports(::Optimizer, ::ConflictStatus)
+    return true
+end
+
+"""
+    ConstraintConflictStatus()
+
+A constraint attribute indicating whether the constraint participates in the last computed conflict.
+"""
+struct ConstraintConflictStatus <: MOI.AbstractConstraintAttribute end
+MOI.is_set_by_optimize(::ConstraintConflictStatus) = true
+
+function MOI.get(model::Optimizer, attribute::ConstraintConflictStatus, index::MOI.ConstraintIndex) 
+    return in(index, model.conflict.rowind)
```

Comment:
This would be clearer as
```julia
function MOI.get(model::Optimizer, ::ConstraintConflictStatus, index::MOI.ConstraintIndex) 
    if model.conflict === nothing
        model.conflict = c_api_getconflict(model.inner)
    end
    return index in model.conflict.rowind
end
```

Reply:
Also, no need to name a variable like `attribute` if it isn't used in the function body.